### ISSUE TITLE
Find Slovakian border guard website links

### ISSUE THREAD

**Issue by EchoRuby68** [Ukraine-Relief-Efforts/scraper#10]

**Comment by clarejor** [comment]
https://www.financnasprava.sk/sk/infoservis/hranicne-priechody

includes wait times at each border crossing

**Comment by Aziroshin** [comment]
> https://www.financnasprava.sk/sk/infoservis/hranicne-priechody
> 
> includes wait times at each border crossing
That link doesn't work anymore. It returns 404 now.

**Comment by Aziroshin** [comment]
- I've overlooked a piece of information in the previous research, and that is that one of the reddit thread opening posts mentioned that `Chop` (railway checkpoint) was closed to individual traffic - I'm not sure whether that refers to some road crossing I haven't seen mentioned anywhere, or whether the rail checkpoint actually isn't available for individuals.
- https://www.stellacentrum.sk/dopravne-informacie/hranicne-priechody/
    - Contains various pieces of information as well as tables with border crossings for the Slovakian neighbour states Austria, Czech Republic, Hungary, Poland and Ukraine, namely the ones corresponding with Uzhhorod and Malyj Bereznyj.
- Another page that seems intended to show waiting times, but... kinda doesn't, I think? (I very much doubt the `00:03` part, for example - and it's always the same, too): https://kordon.customs.gov.ua/uk/home/countries/sk/o
- This page offers quite a wealth of information on the overall subject of crossing into Slovakia and seeking refuge there in general: https://www.ukraineslovakia.sk/en/z-ukrajiny-na-slovensko-english/
- Chop is missing from all of these three pages (Mali Selmentsi is mentioned on ukraineslovakia.sk).

All of this is telling me that Chop probably shouldn't readily be put on display in a way that suggests to people that it's a checkpoint they can just up and head towards. It's probably a good idea to have it in the data somewhere, however.
